nbs/sample_histogram_median.py

```diff
@@ -1,3 +1,4 @@
+# %%
 import numpy as np
```

Comment:
code is stuk, dus file verwijderen?

Reply:
nee is ook voorbeeldcode. Misschien wel netjes om er een werkend stuk code van te maken